src/redux/redux-store.js

```diff
@@ -16,5 +19,6 @@ const store = createStore(
 )
 
 sagaMiddleware.run(watchFetchingSaga)
+sagaMiddleware.run(watchFetchingGallerySaga)
```

Comment:
never do that again.
You should only have one-time **sagaMiddleware.run(rootSaga)**
the inside rootSaga you should have both:
**- watchFetchingSaga
- watchFetchingGallerySaga**